backend/typescript/rest/camperRoutes.ts

```diff
@@ -0,0 +1,39 @@
+import { Router } from "express";
+import { updateCamperDtoValidator } from "../middlewares/validators/camperValidators";
+import CamperService from "../services/implementations/camperService";
+import ICamperService from "../services/interfaces/camperService";
+import { getErrorMessage } from "../utilities/errorUtils";
+
+const camperRouter: Router = Router();
+
+const camperService: ICamperService = new CamperService();
+
+/* Update the camper with the specified camperId */
+camperRouter.put("/:camperId", updateCamperDtoValidator, async (req, res) => {
```

Comment:
we should add a check here that the user has an admin role (only admins should be able to edit camper info), can call isAuthorizedByRole here (like it's done in userRoutes.ts)